Modules/Compatibility/Deprecated/include/itkTreeChangeEvent.h

```diff
@@ -45,6 +45,9 @@ class ITK_TEMPLATE_EXPORT TreeChangeEvent : public ModifiedEvent
   /** Copy constructor */
   TreeChangeEvent(const TreeIteratorBase<TTreeType> & position) { m_ChangePosition = &position; }
 
+  void
+  operator=(const Self &) = delete;
+
```

Comment:
@mseng10 This operator= needs to be placed near the other constructors/destructors at the top of the public section.  For all files where the deleted functions are being moved from "protected/private" they need to be moved to the block wheree existing constructors destructors are.